launch/cli/endpoints.py

```diff
@@ -251,3 +253,44 @@ def edit_endpoint(ctx: click.Context, endpoint_name: str):
         # TODO: Print out a nice error message if the user passes in arguments
         # that fail server-side validation.
         client.edit_model_endpoint(model_endpoint=model_endpoint, **kwargs)
+
+
+@endpoints.command("send")
+@click.argument("endpoint_name")
+@click.option("-r", "--request", help="input request as a json string")
+@click.option("--json-file", help="json file containing request")
```

Comment:
I wonder if we could use `-f` as the short-hand industry convention for specifying a filename